cli/azd/pkg/project/service_target_dotnet_containerapp.go

```diff
@@ -364,7 +366,16 @@ func (_ *containerAppTemplateManifestFuncs) UrlHost(s string) (string, error) {
 
 const infraParametersKey = "infra.parameters."
 
+// Parameter resolves the name of a parameter defined in the ACA yaml definition. The parameter can be mapped to a system
+// environment variable or persisted in the azd environment configuration.
 func (fns *containerAppTemplateManifestFuncs) Parameter(name string) (string, error) {
+	envVarMapping := scaffold.EnvFormat(name)
```

Comment:
@vhvb1989 just a note, not a review comment. I really like this strategy, I think we can think of ways to make this a first-class concept: #4404 in case you have thoughts there.